private4/index.php

```diff
@@ -419,13 +412,13 @@ function lastOne() {
       location.hash = "side" + (data.page + 1);
   if (!location.hash.replace(/^#/, '')) {
   location.hash = "side" + (viewer.currentPage() + 1);
-}    
+}
     });
 
-<!-- Aktiverer separat url (hash) -->
-    viewer.addHandler('open', function() {
-  updateForHash();
-});
+    <!-- Aktiverer separat url (hash) -->
+    viewer.addOnceHandler('open', function() {
```

Comment:
I changed this to `addOnceHandler` because it only needs to run once at the beginning, instead of every time the user opens a different image.